elide-core/src/test/java/com/yahoo/elide/parsers/expression/PermissionToFilterExpressionVisitorTest.java

```diff
@@ -308,6 +356,47 @@ public boolean ok(Model object, RequestScope requestScope, Optional<ChangeSpec>
     static class GOOD8 {
     }
 
+    //Test NOT with "IN" FilterPredicate
+    @Entity
+    @Include
+    @ReadPermission(expression = "NOT (owner is user)")
+    static class GOOD11 {
+    }
+
+    //Test NOT with "LT" FilterPredicate
+    @Entity
+    @Include
+    @ReadPermission(expression = "NOT (less than predicate)")
+    static class GOOD12 {
+    }
+
+    //Test Not with AND FilterExpression
+    @Entity
+    @Include
+    @ReadPermission(expression = "NOT (owner is user AND less than predicate)")
+    static class GOOD13 {
+    }
+
+    //Test Not with OR FilterExpression
+    @Entity
+    @Include
+    @ReadPermission(expression = "NOT (owner is user OR less than predicate)")
+    static class GOOD14 {
+    }
+
+    //Test Not with nested FilterExpression
+    @Entity
+    @Include
+    @ReadPermission(expression = "NOT (owner is user OR (less than predicate AND (NOT owner is user)))")
```

Comment:
A more readable way to do this might be to have two classes in a single test function.  One untransformed - and one transformed.  Extract the filter expressions for both and see if they are identical.


So you would compare the above ReadPermission with:
```"(owner is NOT user AND (greater than equal to predicate OR (owner is user))"```